pkg/ccl/crosscluster/logical/metrics.go

```diff
@@ -82,6 +82,12 @@ var (
 		Measurement: "Events",
 		Unit:        metric.Unit_COUNT,
 	}
+	metaDistSQLReplanCount = metric.Metadata{
```

Comment:
I think you need to actually increment the new metric? Also, could you format the commit message to CRL standards? Eg:

```
crosscluster/logical: add replan metric

This patch adds a new metric blah
```

Reply:
I increment the metric in the following commit, is that what you mean? Or did I miss a setup step.

Thanks, I forget to change the header for the commit. As for adding the whole body, if it's a small enough change do I need to explain all that? i was following your example here: https://github.com/cockroachdb/cockroach/pull/126016/commits